main_tests/production_FMCDAQ3.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Wrapper script for doing a production cycle/routine for fmcdaq3.
+# This script handles
+#
+# Can be called with: ./production_FMCDAQ3.sh
+#
+
+SCRIPT_DIR="$(readlink -f $(dirname $0))"
+
+source $SCRIPT_DIR/lib/production.sh
+source $SCRIPT_DIR/lib/utils.sh
+
+while true; do
+	echo_blue "Please enter your choice: "
+	options=("FMCDAQ3 Test" "Power-Off Carrier" "Power-Off Pi")
+	select opt in "${options[@]}"; do
+		case $REPLY in
+			1)
+				wait_for_board_online
+				export BOARD_SERIAL=$(get_board_serial)
+				echo_blue "Starting FMCDAQ3 Test"
+				production "crr" "$opt" "FMCDAQ3"
```

Comment:
it's outside the scope of this pull request, but for future reference I think the "crr"/target parameter is redundant, i think only talise uses it for something minor. Perhaps it should be removed in future versions